README.md

```diff
@@ -0,0 +1,19 @@
+# ReportPortal Helm Charts
+ReportPortal is a service, it provides great capabilities for speeding up results analysis and reporting by means of built-in analytical features.
+
+### Introduction
+These charts bootstraps a ReportPortal deployment on a Kubernetes cluster using the Helm package manager<br/>
+The repo includes a number of Kubernetes/Helm configurations for installation ReportPortal v4/v5 and aggregator compontent 
+
+### Requirements
+- Kubernetes 1.8+
+- Helm Package Manager
+
+### Usage notes and getting started
+- To start with aggregator installation please follow the guide in ["aggregator" folder](https://github.com/reportportal/kubernetes/tree/master/aggregator)
```

Comment:
aggregator is actually app for internal use and we probably need to explicitly mention it here. 
Just notice users that aggregator is not really needed for them

Reply:
Updated as discussed: Aggregator info has been removed from the root guide